internal/constants/constants.go

```diff
@@ -13,22 +13,29 @@ const (
 	IOSXE     SourceType = "ios-xe"
 )
 
-const DefaultNetboxTagColor = "00add8"
-const DefaultSourceName = "netbox-ssot"
+const SsotTagColor = "00add8"
+const SsotTagName = "netbox-ssot"
+const SsotTagDescription = "Tag used by netbox-ssot to mark devices that are managed by it"
+
+const OrphanTagName = "orphan"
```

Comment:
https://github.com/bb-Ricardo/netbox-sync uses `netbox-synced` and `netbox-synced-orphaned` tags.

`netbox-ssot` seems fine for synced items. Why not using `netbox-ssot-orphans` for orphans items?

It would help users identifying where it comes from.

Thank you!

Reply:
So if I understand correctly just change the following:

```go
const OrphanTagName = "netbox-ssot-orphan"
``

Reply:
> So if I understand correctly, the change you suggest is:

right! thank you!